src/xrCore/Animation/SkeletonMotions.cpp

```diff
@@ -332,7 +332,7 @@ void motions_container::dump()
         sz += it->second->mem_usage();
         Msg("#%3d: [%3d/%5d Kb] - %s", k, it->second->m_dwReference, it->second->mem_usage() / 1024, it->first.c_str());
     }
-    Msg("--- items: %d, mem usage: %d Kb ", container.size(), sz / 1024);
+    Msg("--- items: %uz, mem usage: %uz Kb ", container.size(), sz / 1024);
```

Comment:
The correct usage is to specify parameter in-between `%` and `u`, so it should be `%zu`, not `%uz`.
```suggestion
    Msg("--- items: %uz, mem usage: %zu Kb ", container.size(), sz / 1024);
```

Please, fix similar things in other files.

Reply:
Oh wow, learned something new :)